Update validator group voting docs (#4948)
celo-org/celo-monorepo: packages/docs/celo-holder-guide/voting-validators.md

> # Voting for Validator Groups

Reviewer: Note this is ignored... it takes the title from the SUMMARY.md file. if you change it there, you'll need to also add redirects.

Validators play a critical role in the Celo protocol, determining which transactions get applied and producing new blocks. Selecting organizations that operate well-run infrastructure to perform this role effectively is essential for Celo's long-term success.

> The Celo community makes these decisions by locking CELO tokens and voting for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries that sit between voters and Validators. Every Validator Group has an ordered list of up to 5 candidate Validators. Some organizations may operate a group with their own Validators in it; some may operate a group to which they have added Validators run by others.

Reviewer: ```suggestion
The Celo community makes these decisions by locking CELO and voting for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries that sit between voters and Validators. Every Validator Group has an ordered list of up to 5 candidate Validators. Some organizations may operate a group with their own Validators in it; some may operate a group to which they have added Validators run by others.
```

## Validator Elections

[Validator elections](../celo-codebase/protocol/proof-of-stake/validator-elections.md) are held every epoch (approximately once per day). The protocol elects a maximum of 100 Validators. At each epoch, every elected Validator must be re-elected to continue. Validators are selected [in proportion](../celo-codebase/protocol/proof-of-stake/validator-elections.md#Running-the-Election) to votes received for each Validator Group.

If you hold CELO, or are a beneficiary of a [`ReleaseGold` contract](release-gold.md) that allows voting, you can vote for Validator Groups. A single account can split their LockedGold balance to have outstanding votes for up to 10 groups.

CELO that you lock and use to vote for a group that elects one or more Validators receives [epoch rewards](../celo-codebase/protocol/proof-of-stake/epoch-rewards.md) every epoch (approximately every day) once the community passes a governance proposal enabling rewards. The initial level of rewards is anticipated to be around 6% per annum equivalent (but is subject to change).

> Unlike a number of Proof of Stake protocols, **CELO used for voting is never at risk and will never be [slashed](../glossary.md#slashing)**. The actions of the Validator Groups or Validators you vote for can cause you to receive lower or higher rewards, but the CELO you locked will always be available to be unlocked in the future.

Reviewer: ```suggestion
Unlike a number of Proof of Stake protocols, **CELO used for voting is never at risk**. The actions of the Validator Groups or Validators you vote for can cause you to receive lower or higher rewards, but the CELO you locked will always be available to be unlocked in the future. [Slashing](../glossary.md#slashing) in the Celo protocol applies only to Validators and Validator Groups. 
```

- **Broadens Diversity**: The Celo community aims to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. Support that diversity by considering what new perspectives and strengths the teams you support offer. As well as the backgrounds and experiences of the team, consider that the network security and availability is improved by Validators operating at different network locations, on different platforms, and with different toolchains.

- **Contributes to Celo:** Support Validator Groups that strengthen the Celo developer community, for example through building or operating services for the Celo ecosystem, participating actively in on-chain governance, and answering questions and supporting others, on [Discord](https://chat.celo.org) or the [Forum](https://forum.celo.org).

## The Celo Foundation Voting Policy

> As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their indepedent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.

Reviewer: ```suggestion
As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their independent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.
```

> As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their indepednent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.

Author: ```suggestion
As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their indepedent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.
```

The Celo Foundation has a [Validator Group voting policy](../operations-manual/celo-foundation-voting-policy.md) that it follows when voting with the CELO that it holds. This policy has been developed by the Foundation board and technical advisors with the express goal of promoting the long-term security and decentralization of the network. Validator Groups have an opportunity to apply for Foundation votes every 3 months, and a new cohort is selected based on past performance and contributions. 

You can find the [full set of Validator Groups currently receiving votes, and their addresses linked here](https://docs.google.com/spreadsheets/d/1ltVNkQfXW3lIZxXU52R3IXeD6w21oacWFVb3a-FYRBY/edit?usp=sharing).

## Validator Explorers

> It can be helpful on validator explorers to learn more about validators and validator groups when deciding which group to vote for. to rely on The Celo ecosystem includes a number of great services for browsing registered Validator Groups and Validators.

Reviewer: ```suggestion
The Celo ecosystem includes a number of great services for browsing registered Validator Groups and Validators.
```
